doc/contributing.rst

```diff
@@ -290,7 +290,13 @@ First, you need to have a development environment to be able to build xarray
 Building the documentation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-So how do you build the docs? Navigate to your local
+So how do you build the docs? 
+
+Firstly, make sure you have ``sphinx`` and ``sphinx_rtd_theme`` installed::
+
+    conda install -c anaconda sphinx sphinx_rtd_theme
```

Comment:
This feels a little incongruent with the paragraph beginning with "First" under "Requirements" above

Reply:
Maybe: "In your development environment, install sphinx and sphinx_rtd_theme:"

Reply:
Agreed. I also dropped the question as i felt it was superfluous.